src/CoverCrypt.tsx

```diff
@@ -537,9 +546,12 @@ const CoverCrypt: React.FC<{ kmsToken: string }> = ({ kmsToken }) => {
               {localClearData && <EmployeeTable caption={"Decrypted in browser"} data={localClearData} />}
 
               {/* ENCRYPT/DECRYPT IN KMS */}
-              <HeadingWithCode heading="Encrypt and Decrypt data in KMS" />
+              <HeadingWithCode heading="Encrypt and Decrypt data in the KMS" />
               <Text>
-                The safest implementation is to issue calls to the KMS, whether on-prem or SaaS, to decrypt (or encrypt) the data.
+                The safest implementation is to call the KMS, whether on-prem or secured in the cloud, to decrypt (or encrypt) the data.
+                The key stays within the KMS and is not leaked to the presentation layer; it does however require a call to the KMS for each
```

Comment:
If you prefer `presentation` layer - update the above title with `presentation` instead of `application` ?

Reply:
It is presentation